packages/fast-tooling-react/src/form/form-section.tsx

```diff
@@ -162,50 +157,76 @@ class FormSection extends React.Component<
                 displayValidationInline={this.props.displayValidationInline}
             />
         );
-    }
+    };
 
     /**
      * Renders form items into categories
      */
     private renderCategories(
-        formControlParameters: FormControlParameters[]
+        categories: FormCategoryConfig[] = [],
+        controls: FormControlsWithConfigOptions
     ): React.ReactNode {
-        const categoryParams: FormCategoryProps[] = getCategoryParams(
-            formControlParameters,
-            this.props.orderByPropertyNames
+        return this.getAllCategories(categories).map(
+            (category: FormCategoryConfig, index: number): React.ReactNode => {
+                if (category.title === null) {
+                    return controls.items.map(
+                        (controlItem: FormControlItem): React.ReactNode => {
+                            if (category.items.indexOf(controlItem.propertyName) > -1) {
```

Comment:
`Array.prototype.includes` is a boolean check in the array which might be more suitable for this.